### PR TITLE
Removed 'as e' for Exceptions where the value of 'e' isn't used.

### PR DESCRIPTION
**Pull request by ajeetdsouza**

### CODE REVIEW

File: clients/rospy/src/rospy/impl/tcpros_base.py [hunk 1]
```diff
@@ -744,9 +744,9 @@ def receive_once(self):
         except DeserializationError as e:
             rospyerr(traceback.format_exc())            
             raise TransportException("receive_once[%s]: DeserializationError %s"%(self.name, str(e)))
-        except TransportTerminated as e:
+        except TransportTerminated:
             raise #reraise
```
Comment (mikepurvis) [inline clients/rospy/src/rospy/impl/tcpros_base.py:748]: While you're here, remove the unnecessary comment, please.

**Review by mikepurvis — APPROVED** [review]
Looks like a reasonable change. Was there a static analysis tools which recommended this, or did you manually find the instances of it?

File: clients/rospy/src/rospy/impl/tcpros_base.py (hunk 1 above)
Comment (ajeetdsouza) [reply clients/rospy/src/rospy/impl/tcpros_base.py:748]: Can't, I deleted my fork. Merge this for now, I'll do it from my other PR.

**Comment by ajeetdsouza** [comment]
It was pretty manual. I'm pretty new to open source and I was digging through the source code when I happened to notice one instance of this, so I thought I'd just go through all the results and check. Learned a thing or two along the way 😀

**Comment by mikepurvis** [comment]
Unfortunately, you got one that was needed, as this failing test reveals:

http://build.ros.org/job/Lpr__ros_comm__ubuntu_xenial_amd64/657/testReport/junit/test_rostopic_unit/TestRostopicUnit/test_slicing/

I'd suggest closing this PR, and patching the changes onto a new fork (either check out the repo and cherry pick the commit, or patch it in as https://github.com/ros/ros_comm/commit/014730ea2af8385cdab27ebec99b1c3b1db541d2.patch)

**Comment by ajeetdsouza** [comment]
I'll do that. How do I run tests locally before pushing?

**Comment by mikepurvis** [comment]
```
catkin_make run_tests
```

If you tab-tab, there are a bunch of package-specific test targets if you only need a subset (eg, to retry a failing one).

**Comment by ajeetdsouza** [comment]
Alright, thanks!

**Comment by ajeetdsouza** [comment]
Fixed in https://github.com/ros/ros_comm/pull/1278

**Comment by dirk-thomas** [comment]
@ajeetdsouza Looking at the merged PR #1278 the issue described in this ticket wasn't addressed by it. Please consider to create a pull request any unnecessary `as e` uses.

**Pull request closed without merging**